jbpm-bpmn2/src/test/resources/BPMN2-EndEventSignalWithData.bpmn2

```diff
@@ -4,8 +4,8 @@
   <bpmn2:process id="src.simpleEndSignal" drools:packageName="com.myteam.testit" drools:version="1.0" name="simpleEndSignal" isExecutable="true">
     <bpmn2:startEvent id="processStartEvent" drools:selectable="true" color:background-color="#9acd32" color:border-color="#000000" color:color="#000000" name="">
       <bpmn2:extensionElements>
-        <drools:metaData name="elementname">
-          <drools:metaValue><![CDATA[]]></drools:metaValue>
+        <drools:metaData name="customScope">
```

Comment:
As it seems you are going on further improvements already (modeling operations at bpmn process level changes this for).
implementation
and the value should be  
##Kafka

to be align with the spec

Reply:
ok, done